applications/osb-portal/src/components/App.tsx

```diff
@@ -8,7 +10,7 @@ import SentryErrorBoundary from "./sentry/SentryErrorBoundary";
 
 import { Latest } from "./latest/Latest";
 
-import { Header, Banner, WorkspaceDrawer, Workspaces, WorkspaceToolBox, ErrorDialog } from "./index";
+import { Header, Banner, WorkspaceDrawer, Workspaces, WorkspaceToolBox, ErrorDialog, NWBExplorerFrame } from "./index";
```

Comment:
Should be generic. Depending on the type of the workspace will show NWB explorer, NEtpyne or  a bare notebook application. A name Like WorkspaceFrame would suit better.

Reply:
Understand, question: the url setup is specific for NWB Explorer and the iFrame is generic. I mean: we would like to have an iFrame component where we can pass in the url. Right?